lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3InputStream.java

```diff
@@ -243,6 +243,7 @@ private void closeStream()
 
         try (var _ = in) {
             in.abort();
+            in.release();
```

Comment:
can you add the reference document (in the commit message) where it was mentioned this is recommended?